### PR TITLE
Edit dev guide 

### PR DESCRIPTION
**Pull request by ph-nathan**
What have you done in this PR?
- Edit all instances of AddressBook to CcaCommander in DG
- Add general model diagram 
- Update Ui diagram 
- Update all diagrams with previous instances of AddressBook
*

Dev Testing Steps / Screenshots

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -444,18 +442,18 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
     Use case ends.
 
-<ins>**Use case: UC09 - Add member to an event**</ins>
+<ins>**Use case: UC09 - Enrol member to an event**</ins>
 
-**Guarantees: MSS -> Specified member will be added to the event.**
+**Guarantees: MSS -> Specified member will be enroled to the event.**
```
Comment (dylkaw) [inline docs/DeveloperGuide.md]:
```suggestion
**Guarantees: MSS -> Specified member will be enrolled to the event.**
```
Minor typo here.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -475,18 +473,18 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
-<ins>**Use case: UC10 - Delete member from an event**</ins>
+<ins>**Use case: UC10 - Unenrol member from an event**</ins>
 
-**Guarantees: MSS -> Specified member will be deleted from the event.**
+**Guarantees: MSS -> Specified member will be unenroled from the event.**
```
Comment (dylkaw) [inline docs/DeveloperGuide.md]:
```suggestion
**Guarantees: MSS -> Specified member will be unenrolled from the event.**
```
Minor typo here.

File: docs/images/BetterModelClassDiagram.png
Comment (dylkaw) [inline docs/images/BetterModelClassDiagram.png:1]: Should 'Person' be changed to 'Member', with additional attribute 'Gender'?

File: docs/images/ModelClassDiagram.png
Comment (dylkaw) [inline docs/images/ModelClassDiagram.png:1]: Should the boxes for `Member`, `Event` and `Attendance` be standardised?

File: docs/images/StorageClassDiagram.png
Comment (dylkaw) [inline docs/images/StorageClassDiagram.png:1]: Should JsonAdaptedPerson be JsonAdaptedMember?

**Review by dylkaw — CHANGES_REQUESTED** [review]
Some comments on replacing AB3 object names with CCACommander object names.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -68,41 +68,41 @@ The sections below give more details of each component.
...
 
-The `UI` component uses the JavaFx UI framework. The layout of these UI parts are defined in matching `.fxml` files that are in the `src/main/resources/view` folder. For example, the layout of the [`MainWindow`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/ui/MainWindow.java) is specified in [`MainWindow.fxml`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/resources/view/MainWindow.fxml)
+The `UI` component uses the JavaFx UI framework. The layout of these UI parts are defined in matching `.fxml` files that are in the `src/main/resources/view` folder. For example, the layout of the [`MainWindow`](https://github.com/AY2324S1-CS2103T-F11-1/tp/blob/master/src/main/java/seedu/ccacommander/ui/MainWindow.java) is specified in [`MainWindow.fxml`](https://github.com/AY2324S1-CS2103T-F11-1/tp/blob/master/src/main/resources/view/MainWindow.fxml)
 
-The `UI` component,
+The `UI` component
```
Comment (wr1159) [inline docs/DeveloperGuide.md]: Just wondering, why did you remove the comma here?

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -68,41 +68,41 @@ The sections below give more details of each component.
...
 
 * executes user commands using the `Logic` component.
 * listens for changes to `Model` data so that the UI can be updated with the modified data.
 * keeps a reference to the `Logic` component, because the `UI` relies on the `Logic` to execute commands.
-* depends on some classes in the `Model` component, as it displays `Person` object residing in the `Model`.
+
```
Comment (wr1159) [inline docs/DeveloperGuide.md]: Does it no longer depend on classes in the `Model` Component?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (wr1159) [reply docs/DeveloperGuide.md]: Oh right, enrolled is the correct spelling

File: docs/images/ModelClassDiagram.png
Comment (wr1159) [inline docs/images/ModelClassDiagram.png:1]: Nit but should we standardise the styling of Member, Event, Attendance with the other classes?

**Review by wr1159 — COMMENTED** [review]
Added a few comments.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -282,7 +280,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | power user    | delete an event                             | I can remove it from the schedule                                                 |
 | `* * *`  | power user    | view a list of all events hosted by the CCA | I can see event history                                                           |
 | `* * *`  | beginner user | add a member to an event                    | I can track which members are participating in the event                          |
-| `* * *`  | beginner user | remove a member from an event               | I can amend adding the wrong person to an event                                   |
+| `* * *`  | beginner user | remove a member from an event               | I can amend adding the wrong  member to an event                                   |
```
Comment (andrechuakj) [inline docs/DeveloperGuide.md]: I think there is an additional space here
```suggestion
| `* * *`  | beginner user | remove a member from an event               | I can amend adding the wrong member to an event                                   |
```

**Review by andrechuakj — COMMENTED** [review]
Just a nit, the rest LGTM

**Review by dylkaw — APPROVED** [review]
